ui/ozone/platform/starboard/surface_factory_starboard.cc

```diff
@@ -16,8 +16,27 @@
 
 namespace ui {
 
-SurfaceFactoryStarboard::SurfaceFactoryStarboard() {}
+SurfaceFactoryStarboard::SurfaceFactoryStarboard() {
+  egl_implementation_ = std::make_unique<GLOzoneEGLStarboard>();
+}
```

Comment:
With the code as is, `egl_implementation_` is initialized twice, once using a no-arguments constructor, then by l.20 (see Scott Meyer's Effective C++: 55 Specific Ways to Improve Your Programs and Designs -- "Item 12 prefer initialization to assignment in constructors"). Instead, create it in the initialization list.

Another question I have is why a `unique_ptr<>`? Could it be a normal variable and then return its address in l.37 ?

Reply:
Done.